src/components/picker.tsx

```diff
@@ -56,25 +56,37 @@ export default class Picker extends React.Component<Props, State> {
     this.props.onExit()
   }
 
-  renderItems(data: Record[]) {
-    const { columns, multiselect } = this.props
-    const { mode, picked, search } = this.state
+  renderEmpty(fetching: Boolean) {
```

Comment:
I'm actually really curious your thought on this.  I always see `renderSomething` methods as a bit of an anti-pattern, preferring to make this SFC instead, even if it's in the same file.  Is there a good reason not to do that in this case?  I'm just thinking about ease of reuse down the line.

Reply:
Good call. It should be.